src/main/PlagueMain.java

```diff
@@ -638,46 +641,46 @@ public void registerClientCommands(CommandHandler handler) {
                         return;
                     }
 
-                    CustomPlayer cPly = uuidMapping.get(player.uuid());
-                    PlagueTeam pTeam = teams.get(cPly.team);
+                    // Same as the variable player, but converted into a CustomPlayer for additional plague features.
+                    CustomPlayer plaguePlayer = uuidMapping.get(player.uuid());
+                    PlagueTeam playerTeam = teams.get(plaguePlayer.team);
 
-                    if (pTeam.leader.player.uuid() != cPly.player.uuid()) {
+                    // Checks if caller is the leader, exits if not (insufficient permission)
+                    if (!Objects.equals(playerTeam.leader.player.uuid(), plaguePlayer.player.uuid())) {
                         player.sendMessage("[accent]You must be team leader to kick players!");
                         return;
                     }
 
                     if (args.length != 0) {
-                        for (CustomPlayer other : pTeam.players) {
-                            if (("" + other.player.id()).equals(args[0]) ||
-                                    other.player.name.equalsIgnoreCase(args[0]) ||
-                                    other.rawName.equalsIgnoreCase(args[0])) {
-                                if (other.player == player)
+                        for (CustomPlayer teammate : playerTeam.players) {
```

Comment:
case

Reply:
I don't see how this is an issue, seems readable enough to me

Reply:
`teamMate`
even if your not in java `teammate` is invalid casing

Reply:
teammate is one word